tables/output_tables.Rmd

```diff
@@ -143,19 +143,20 @@ soi_df <- tmb_hist %>%
     ),
     # add Lynch syndrome
     cancer_predispositions = case_when(
-      Kids_First_Biospecimen_ID == "BS_8AY2GM4G" ~ "NF-1, Lynch syndrome",
+      Kids_First_Biospecimen_ID == "BS_8AY2GM4G" ~ "Lynch syndrome",
+      sample_id == "7316-515" ~ "Other inherited conditions NOS",
       sample_id == "7316-3058" ~ "Lynch Syndrome",
       sample_id == "7316-2189" ~ "Lynch Syndrome",
       Kids_First_Biospecimen_ID == "BS_85Q5P8GF" ~ "Lynch syndrome",
       TRUE ~ as.character(cancer_predispositions)),
     # add pathogenic germline variants
     `Pathogenic germline variant` = case_when(
-      Kids_First_Participant_ID == "PT_3CHB9PK5" ~ "MSH6",
-      Kids_First_Participant_ID == "PT_0SPKM4S8" ~ "PMS2",
+      Kids_First_Participant_ID == "PT_3CHB9PK5" ~ "MSH6 NM_000179.2 c.3439-2A>G splice (P)",
+      Kids_First_Participant_ID == "PT_0SPKM4S8" ~ "PMS2 p.S46I (LP)",
```

Comment:
I don't see this variant in the spreadsheet you shared has any amino acid change. I just see `c.137G>T` at nucleotide level.

Reply:
Ah one of these was missing and a collaborator found the protein change. I'll get back to you

Reply:
Also noted below, column BO in the shared excel sheet:

>PMS2:NM_000535:exon2:c.G137T:p.S46I,PMS2:NM_001322006:exon2:c.G137T:p.S46I,PMS2:NM_001322014:exon2:c.G137T:p.S46I